recipes/python-mip/activate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -z $PMIP_CBC_LIBRARY ]; then
+    export CONDA_BACKUP_PMIP_CBC_LIBRARY=$PMIP_CBC_LIBRARY
+fi
+
+export PMIP_CBC_LIBRARY=$PREFIX/lib/libCbc${SHLIB_EXT}
```

Comment:
```suggestion
export PMIP_CBC_LIBRARY=$CONDA_PREFIX/lib/libCbc${SHLIB_EXT}
```
`$PREFIX` and `%LIBRARY_PREFIX%` are the temporary build/install environments used by conda-build, but on the end-user machine, the package will be installed into an environment of their choice. This user-environment is `$CONDA_PREFIX`.

Reply:
Not sure how your comment here passed under my radar - thanks!